### PR TITLE
K8SPXC-167 Add proxysql syncusers

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: e2e-tests/build [hunk 1]
```diff
@@ -28,6 +28,8 @@ build_operator() {
             -ldflags '-w -s -X main.GitCommit=${GIT_COMMIT} -X main.GitBranch=${GIT_BRANCH}' \
             -o build/_output/bin/user-manager \
             cmd/user-manager/main.go
+        cd /go/src/${package}/build/_output/bin
+        git clone https://github.com/percona/proxysql-admin-tool
```
Comment (delgod) [inline e2e-tests/build]: we can add this repo into vendor dir and copy tool from it during the build

File: cmd/user-manager/proxy/proxy.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package proxy
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func SyncUsers() error {
```
Comment (delgod) [inline cmd/user-manager/proxy/proxy.go]: what if proxysql is disabled?

File: cmd/user-manager/proxy/proxy.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	proxyUser := "root"
+	proxyPass := os.Getenv("MYSQL_ROOT_PASSWORD")
+	proxyHost := os.Getenv("PROXY_SERICE")
+	proxyPort := "3306"
+	clusterUser := "root"
+	clusterPass := os.Getenv("MYSQL_ROOT_PASSWORD")
+	clusterHost := os.Getenv("PXC_SERICE")
+
+	pat := exec.Command("bash", "proxysql-admin",
```
Comment (delgod) [inline cmd/user-manager/proxy/proxy.go]: I think bash not needed if the file copied correctly
```suggestion
	pat := exec.Command("proxysql-admin",
```

File: build/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,53 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal
-RUN microdnf update && microdnf clean all
+#FROM registry.access.redhat.com/ubi7/ubi-minimal
```
Comment (delgod) [inline build/Dockerfile]: we need to use ubi image in all operator images, identical to ProxySQL image https://github.com/percona/percona-docker/blob/master/proxysql/Dockerfile.k8s

File: build/Dockerfile [hunk 5]
```diff
@@ -1,5 +1,53 @@
...
+                hostname \
+                perl-DBD-MySQL \
+                perl-DBI \
+        && yum clean all
+
+#RUN yum install -y gnu-getopt
+
+RUN groupadd -g 1001 proxysql \
+    && useradd -u 1001 -r -g 1001 -s /sbin/nologin \
+        -c "Default Application User" proxysql
```
Comment (delgod) [inline build/Dockerfile]: do we use `proxysql` user?

**Comment by delgod** [comment]
discussed offline, should be reworked

**Pull request closed without merging**